tests/test_commodity_interfaces.py

```diff
@@ -2,100 +2,18 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from matplotlib.patches import Patch
+from  data_adapter.preprocessing import Structure
 
-# Define filename to analyse
-name = 'sedos_set_io'
 
-# choose option
-# com_choice = 'secondary'
-com_choice = 'all'
 
-# Read the Excel file and process_set sheet
-df = pd.read_excel(name+'.xlsx', sheet_name='Process_Set')
+structure = Structure(
+    "SEDOS_Modellstruktur",
+    process_sheet="Processes_O1",
+    parameter_sheet="Parameter_O1",
+    helper_sheet="Helper_O1",
+)
 
-# Fill empty cells to facilitate logic - it is not considered as it has not one of the predefined category codes
-df.fillna('dummytext', inplace=True)
+structure.plot_commodity_interfaces()
 
-# Filter out processes which impair the logic
-df = df[~df['process'].str.contains('storage|transport|import|export')]
-
-# Delete square brackets from dataframe
-df['input'] = df['input'].str.replace(r'[][]', '', regex=True)
```

Comment:
In my last review I additionally deleted blank spaces to avoid unintended double naming and the _orig ending for commodities to simplify the graph.

Reply:
df['input'] = df['input'].str.replace(r' ', '', regex=True)
df['output'] = df['output'].str.replace(r' ', '', regex=True)
df['input'] = df['input'].str.replace(r'_orig', '', regex=True)
df['output'] = df['output'].str.replace(r'_orig', '', regex=True)

Reply:
I think it might be even neccessary to see the blank spaces. Since I am using the data_adapter itself we have to have a consistent naming here and might miss an opportunity to identify potential typos.

Reply:
I don't understand this point. Could you elaborate this more please?
We are talking about the naming of inputs and outputs, which might be separated with commas, right?

Reply:
Yes but the whitespace trimming already happens within the data_adapter beforehand, as the plot function is now part of the class I thought it is better to reuse already existing parts of code 
To improve consistency I want to use exactly what is going to be input for the data_adapter_oemof.